themes/theme-gmd/pages/Product/components/Characteristics/Swatch/index.jsx

```diff
@@ -98,6 +98,7 @@ class Swatch extends PureComponent {
             testId={`productSwatch.${id}`}
             swatch={swatch}
             onClick={this.handleItemSelection}
+            widgetId="@shopgate/engage/product/VariantSwatch"
```

Comment:
The widget should be defined by the used tag instead of an attribute. In this case it should use a widget-component called something like `VariantSwatch` and not pass a `widgetId` to it.